trace_decoder/src/lib.rs

```diff
@@ -472,9 +463,9 @@ mod hex {
     pub fn deserialize<'de, D: Deserializer<'de>, T>(deserializer: D) -> Result<T, D::Error>
     where
         T: hex::FromHex,
-        T::Error: fmt::Display,
+        T::Error: std::fmt::Display,
     {
-        let s = Cow::<str>::deserialize(deserializer)?;
+        let s = String::deserialize(deserializer)?;
```

Comment:
This looked like zero-copy deserialization, but actually isn't.
I don't particularly care about perf, so just replaced with simple code.

Reply:
> I don't particularly care about perf

Although I would agree in this context, I'd tend to simply advise caution in general regarding this. Even if the decoder in itself is not the bottleneck, I think it would be nice to keep it, to some extent, as performant as we can, at least while keeping a relatively simple code :)